xchange-kraken/pom.xml

```diff
@@ -29,6 +29,10 @@
             <artifactId>xchange-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-jre8</artifactId>
```

Comment:
make this dependancy last and add `<scope>test</scope>`

Reply:
Okay, thanks.